cylc/flow/network/schema.py

```diff
@@ -1182,4 +1194,17 @@ class Mutations(ObjectType):
         description=TaskActions._meta.description)
 
 
-schema = Schema(query=Queries, mutation=Mutations)
+# ** Subscription Related ** #
+
+class Subscriptions(ObjectType):
+    class Meta:
+        description = """Multi-Workflow root level subscriptions."""
+    workflows = List(
+        Workflow,
+        description=Workflow._meta.description,
+        ids=List(ID, default_value=[]),
+        exids=List(ID, default_value=[]),
+        resolver=subscribe_workflows)
```

Comment:
This is pretty much the same as the `class Queries` we have already for a Query. Only difference being the resolver...

Reply:
This duplication is ok..